pages/getting-started/developers.md

```diff
@@ -122,14 +122,55 @@ You can also email us directly at uswebdesignstandards@gsa.gov.
 * Uses a **[modified BEM](https://pages.18f.gov/frontend/css-coding-styleguide/naming/)** approach created by 18F for naming CSS selectors. Objects in CSS are separated by single dashes. Multi-word objects are separated by an underscore (For example: `.usa-button-cool_feature-active`).
 * Uses **modular CSS** for scalable, modular, and flexible code.
 * Uses **nesting** when appropriate. Nest minimally with up to two levels of nesting.
-* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying the `_variables.scss` file into your own project's Sass folder, changing applicable variable values, and importing it before uswds in Sass.
-* Hard-coded magic numbers are avoided and, if necessary, defined in the `_defaults.scss` file.
+* Hard-coded magic numbers are avoided and, if necessary, defined in the `core/variables` scss file.
 * Media queries are built **mobile first**.
 * **Spacing units** are as much as possible defined as rem or em units so they scale appropriately with text size. Pixels can be used for detail work and should not exceed 5px (For example: 3px borders).
 
 **For more information, visit:
 [https://pages.18f.gov/frontend/css-coding-styleguide/](https://pages.18f.gov/frontend/css-coding-styleguide/)**
 
+
+## Customization/theming
+
+The standards are ready to be customized to use different typography, colors and grid system. The easiest way to customize the standards is to use Sass and override the standards global variables. If it isn't possible to use Sass, theming must be done by over-riding CSS rules the standards set.
+
+To start theming through Sass, copy the `core/variables` file into your own project's Sass folder, changing applicable variable values, and importing it before the WDS. Below is an example of customizing the WDS.
+
+```scss
+// src/main.scss
+@import 'src/my-custom-vars';
+@import 'lib/uswds/src/stylesheets/all';
+```
+
+```scss
+// src/_my-custom_vars.scss
+
+// Colors
+$color-primary: #2c3e50;
+$color-secondary: #ad2020;
+$color-secondary-dark: #b0392e;
+
+// Typography
+$font-serif: 'Georgia', 'Times', serif;
+$h2-font-size: 2rem;
+$h3-font-size: 1.75rem;
+$heading-line-height: 1.4;
+
+// Grid/breakpoints
+$small-screen:  540px !default;
+$medium-screen: 620px !default;
+$large-screen:  1120px !default;
+```
+
+Do not every change any code in your `uswds` folder. Doing so could make it impossible to upgrade WDS in the future without undoing your changes.
```

Comment:
Hrmmm... not entirely sure we need to be explicit about this, but it could not hurt. Maybe change to:

> NOTE: If you plan on upgrading to newer versions of the Standards in the future, or are not using your own forked version of the Standards, try to avoid making changes in the Standards folder themselves. Doing so could make it impossible to upgrade in the future without undoing your custom changes.